pyproject.toml

```diff
@@ -11,12 +11,12 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "otx"
 dynamic = ["version"]
-description = "OpenVINO™ Training Extensions: Train, Evaluate, Optimize, Deploy Computer Vision Models via OpenVINO™"
+description = "OpenVINO Training Extensions: Train, Evaluate, Optimize, Deploy Computer Vision Models via OpenVINO"
```

Comment:
Why is this removed?

Reply:
@vinnamkim from some dev environment (e.g., WSL), pre-commit (ruff) complained about this file while decoding to utf-8. it seems to be passed in my current environment but i think TM is not important thing to the project description so i've removed it. is there any concerns about it?

Reply:
![image](https://github.com/openvinotoolkit/training_extensions/assets/26541465/2f679709-9eab-4d1a-82da-e24aadeefc9c)
The trademark symbol is definitely present at least in sources provided by Intel. I don't think it is good idea to drop it.

Reply:
This is the error that i've met before - https://github.com/openvinotoolkit/training_extensions/actions/runs/8277980414/job/22649413049
Anyway, I will try to check again with adding ™ to the project description.